### ISSUE TITLE
unable to use my classifiers.tflite and label.txt. Please help

### ISSUE THREAD

**Issue by AdarshJain2305** [tensorflow/tensorflow#59807]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1.  It must be a bug, a feature request, or a significant problem with the
    documentation (for small docs fixes please send a PR instead).
2.  The form below must be filled out.
3.  It shouldn't be a TensorBoard issue. Those go
    [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information

-   **Have I written custom code (as opposed to using a stock example script
    provided in TensorFlow)**:
-   **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
-   **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue
    happens on a mobile device**:
-   **TensorFlow installed from (source or binary)**:
-   **TensorFlow version (use command below)**:
-   **Python version**:
-   **Bazel version (if compiling from source)**:
-   **GCC/Compiler version (if compiling from source)**:
-   **CUDA/cuDNN version**:
-   **GPU model and memory**:
-   **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with:

```bash
python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"
```

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by tilakrayal** [comment]
@AdarshJain2305,
TensorFlow Lite models may come with different associated files. For example, natural language models usually have vocab files that map word pieces to word IDs; classification models may have label files that indicate object categories. Without the associated files (if there are), a model will not function well.

The associated files can now be bundled with the model through the metadata Python library. The new TensorFlow Lite model becomes a zip file that contains both the model and the associated files. It can be unpacked with common zip tools. This new model format keeps using the same file extension, .tflite. It is compatible with existing TFLite framework and Interpreter. 
Also please take a look at this official doc link for the [reference](https://www.tensorflow.org/lite/models/convert/metadata#label_output).

https://www.tensorflow.org/lite/models/convert/metadata#adding_metadata_using_flatbuffers_python_api

Thank you!

**Comment by AdarshJain2305** [comment]
@tilakrayal 
in which part of the pose classification do I need make changes to use my classifiers.tflite and label.txt files

**Comment by AdarshJain2305** [comment]
*pose classification folder

**Comment by tilakrayal** [comment]
@AdarshJain2305,

- Preprocess the pose classification training data into a CSV file that specifies the landmarks (body keypoints) detected by the MoveNet model, along with the ground truth pose labels.

- Build and train a pose classification model that takes the landmark coordinates from the CSV file as input, and outputs the predicted labels.

- Convert the pose classification model to TFLite.

The detailed explanation was available at this official document **pose_classification.ipynb** for reference.
https://github.com/tensorflow/tensorflow/blob/2823f9082cc69174bab5f5478999258139de8610/tensorflow/lite/g3doc/tutorials/pose_classification.ipynb

**Comment by Cranky-cat** [comment]
What is the label.txt file format? Is there a case?

**Comment by Cranky-cat** [comment]
@tilakrayal

**Comment by Cranky-cat** [comment]
> 


@tilakrayal What is the label.txt file format? Is there a case?

**Comment by tilakrayal** [comment]
@Cranky-cat,
Looks like the similar issue was raised https://github.com/tensorflow/tensorflow/issues/59968. We can track the issue there. Thank you!
